src/jvm/main/org/jetbrains/kotlinx/lincheck/Result.kt

```diff
@@ -64,63 +54,55 @@ class ExceptionResult private constructor(
      * Exception is stored to print it's stackTrace in case of incorrect results
      */
     val throwable: Throwable,
-    override val wasSuspended: Boolean,
     /**
-     * Normalized version of the exception class
+     * Exception class canonical name
      */
-    tClazz: Class<out Throwable>,
-) : Result() {
+    internal val tClassCanonicalName: String
```

Comment:
You don't need to pass it to the constructor. If needed, the class name is accessible from `throwable`.